src/components/navigation/header/navigationTenantMenu/index.tsx

```diff
@@ -46,6 +46,8 @@ const NavigationTenantMenu: FC<Props> = ({ user, selectTenant }) => {
         onClose={onClose}
         isOpen={isOpen}
         initialFocusRef={initialFocusRef}
+        isLazy={true}
```

Comment:
Using lazy here because of `fit-content` width that breaks on mobile. This menu is not visible on mobile